coq/coq-syntax.el

```diff
@@ -488,6 +488,10 @@ so for the following reasons:
     ("Instance" nil "Instance #:#.\nProof.\n#Defined." t "Instance")
     ("Program Instance" "pinstance" "Program Instance [ # ] => # where \n# := #;\n# := #." t "Program\\s-+Instance")
     ("Let" "Let" "Let # : # := #." t "Let")
+    ("Local Ltac2" nil "Local Ltac2 # := #" t "Local\\s-+Ltac2")
+    ("Ltac2 Type" "ltac2 ty" "Ltac2 Type # := #" t "Ltac2 Type")
+    ("Ltac2 Type" "ltac2 oty" "Ltac2 Type # ::= #" t "Ltac2 Type")
+    ("Ltac2" "ltac2" "Ltac2 # := #" t "Ltac2")
```

Comment:
Two details:

* You might want to add a final dot after the last `#`s.

* The abbreviations you choose shouldn't contain a space (currently `ltac2 ty` etc. don't work, cf. `M-x abbrev-mode RET ltac2 SPC`)

```suggestion
    ("Local Ltac2" nil "Local Ltac2 # := #." t "Local\\s-+Ltac2")
    ("Ltac2 Type" "ltac2ty" "Ltac2 Type # := #." t "Ltac2 Type")
    ("Ltac2 Type" "ltac2oty" "Ltac2 Type # ::= #." t "Ltac2 Type")
    ("Ltac2" "ltac2" "Ltac2 # := #." t "Ltac2")
```

Reply:
On second thought, I suggest that you replace all `ltac2…` abbreviations (the 2nd string arguments) with `lt2…`.

Otherwise, when one enable `M-x abbrev-mode`, we cannot write this anymore:

`From Ltac2 Require Import Ltac2.`

and get

`From  Ltac2 # := # Require Import Ltac2.`